src/EventSourcingAnalyzer.php

```diff
@@ -16,5 +16,19 @@ public function commandMap(): VertexMap;
 
     public function eventMap(): VertexMap;
 
-    public function aggregateMap(): AggregateConnectionMap;
```

Comment:
Is replaced by `\EventEngine\InspectioGraph\Analyzer\AggregateConnectionAnalyzer::aggregateConnectionMap()`.